src/scores/processing/isoreg_impl.py

```diff
@@ -503,10 +503,13 @@ def _contiguous_mean_ir(
 ) -> np.ndarray:
     """
     Performs classical (i.e. for mean functional) contiguous quantile IR on tidied data x, y.
-    Uses sklearn implementation rather than supplying the mean solver function to `_contiguous_ir`,
```

Comment:
Thanks for this update @AJTheDataGuy!
The original motivation for using the sklearn version rather than our independent implementation was the speed improvement (the scipy version didn't exist at the time). Is it possible to do a speed comparison to understand the possible impact on performance based on this change?